README.md

```diff
@@ -85,6 +91,8 @@ The pipeline name starts with `PokyAmiPipeline-` in the CodePipeline page.
 
 Example stack name: PokyAmiPipeline
 
+Expected build time: 52min / rebuild: 17min
```

Comment:
Could you describe what you mean by rebuild? If the rebuild is also from scratch.. :)

Reply:
done